tests/ballerina-compiler-api-test/src/test/resources/test-src/annotations_in_tests_project/Ballerina.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+org = "testorg"
+name = "annotations_in_tests_project"
+version = "0.1.0"
+distribution = "2201.10.1"
```

Comment:
we dont need to restrict this to 2201.10.1 right?

Reply:
Ack. Will remove